### PR TITLE
Update CONCOCT_CONCOCT process label

### PR DESCRIPTION
**Pull request by prototaxites**
Concoct runs very slowly (days) without additional threads - bump the default number of CPUs by changing the label to `process_high` to improve runtimes.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by prototaxites** [comment]
Found bug with concoct conda test - bioconda pulls latest version of sklearn, but this causes concoct to fail. Fixed the conda recipe, but am creating a multi-package container with the same fixed version so the container it if it's rebuilt.

https://github.com/BioContainers/multi-package-containers/pull/2687

**Review by jfy133 — COMMENTED** [review]
What is the purpsoe of `scikit-learn`?

**Comment by prototaxites** [comment]
Concoct is finally fixed in Bioconda - should be good to go assuming this passes!

**Review by jfy133 — APPROVED** [review]
